hyperliquid/info.py

```diff
@@ -53,6 +53,55 @@ def user_state(self, address: str) -> Any:
                 }
         """
         return self.post("/info", {"type": "clearinghouseState", "user": address})
+    
+    def user_fees(self, address: str) -> Any:
```

Comment:
nit: could you move this below candles_snapshot. I don't think it should be so high within this file